src/utils/utils.js

```diff
@@ -11,13 +11,19 @@ class Utils {
   * Returns the path where the data directory is, and also creates the directory if it doesn't exist.
   */
   static getDataDir() {
+    const osDataDir = app.getPath('userData');
+
     let dataDir;
     if (_.indexOf(process.argv, '--dev') === -1) {
+      const version = Config.CONTRACT_VERSION_NUM;
+      const testnet = Config.TESTNET;
+      const pathPrefix = testnet ? 'testnet' : 'mainnet';
+
       // production
-      dataDir = `${app.getPath('userData')}/${version}`;
+      dataDir = `${osDataDir}/${pathPrefix}${version}`;
```

Comment:
missing a `slash` between `${pathPrefix}` and `${version}`?

Reply:
Actually I intended it this way. I'll add a slash though. It makes more sense heirarchy-wise.